SSZipArchive/Common.h

```diff
@@ -0,0 +1,81 @@
+#ifndef SSZipCommon
```

Comment:
Added the README here. https://github.com/Microsoft/react-native-code-push/blob/asset-update/SSZipArchive/README.md